### PR TITLE
feat(content): Short Rulei Mission - Umbral Reach

### PR DESCRIPTION
**Pull request by Saugia**
**Content (Artwork / Missions / Jobs)**

## Summary
This PR adds a short-and-sweet mission where the vyu-Ir reacts to you visting the area above Kanguwa/G-3191. The area's official name is the "Umbral Reach", which will be given a map label in a future mission.

The mission offers on the condition you have both met the Rulei and have travelled through L-118. It is possible to visit the Reach before meeting the vyu-Ir, but I think it's fine that the conversation still only happens after you meet it and then (re)visit the Reach. As invisible missions only work after landing, it's possible to visit L-118 but not come back to the vyu-Ir until much later to get this mission, unless you land on one of the Rulei worlds. I think this is fine too, though.

This PR uses the replacement names for the vyu-Ir and the first contact mission introduced in #8594, so either PR will adjust to whichever is merged first.

## Save File
Just fly to the respective areas with a JD.

### CODE REVIEW

**Comment by Saugia** [comment]
After some second thought, I decided to cut the G-3191 mission. Players will easily jump through the system while exploring, and since it's a binary supergiant system, the likeliness of them flying to the center to see the structure is low unless someone explicitly tells them it's there or they have their sound on/are listening for it. Perhaps in the future it will have a mission, but for now I think it's fine it doesn't, even if it's a tad odd.

**Comment by Arachi-Lover** [comment]
Maybe add some `...` or such to break within the phrase and keep more of the thing where the Rulei can only barely communicate? Given previously it's only been single words or "An end" a three-word phrase right away on this short mission striked me a bit odd.

File: data/rulei/rulei.txt [hunk 1]
```diff
@@ -34,7 +34,6 @@ mission "First Contact: Rulei"
 	on offer
 		fail "Ira Contact"
 	on enter "Kanguwa"
-		fail
```
Comment (Anarchist2) [inline data/rulei/rulei.txt:37]: Why is this removed?

File: data/rulei/rulei.txt [hunk 2]
```diff
@@ -90,6 +89,24 @@ mission "Rulei Planet"
...
+	landing
+	to complete
+		never
+	to offer
+		has "First Contact: Rulei: offered"
+		has "visited system: L-118"
+	on enter "Kanguwa"
+		log "Factions" "Rulei" `The ancient Rulei ship stated to be careful of "Umbral's Reach".`
+		conversation
+			`Out of nowhere, the mysterious whispers of the Rulei begin to seep into your mind once more. They murmur slowly at first, but grow faster and louder from the direcetion of the ancient vessel, eventually coalescing into one spoken phrase.`
```
Comment (Anarchist2) [inline data/rulei/rulei.txt:104]:
```suggestion
			`Out of nowhere, the mysterious whispers of the Rulei seep into your mind once more. They murmur slowly at first, but grow faster and louder from the direction of the ancient vessel, eventually coalescing into one spoken phrase.`
```

File: data/rulei/rulei.txt [hunk 3]
```diff
@@ -90,6 +89,24 @@ mission "Rulei Planet"
...
+mission "Umbral's Reach"
+	invisible
+	landing
+	to complete
+		never
+	to offer
+		has "First Contact: Rulei: offered"
+		has "visited system: L-118"
+	on enter "Kanguwa"
+		log "Factions" "Rulei" `The ancient Rulei ship stated to be careful of "Umbral's Reach".`
```
Comment (Anarchist2) [inline data/rulei/rulei.txt:102]: I don't think this is worth logging. See the Archon nanobots logbook entry: it doesn't mention to avoid Sayaiban, but it instead logs the insight behind the Archon's actions.

File: data/rulei/rulei.txt [hunk 4]
```diff
@@ -90,6 +89,24 @@ mission "Rulei Planet"
...
+		conversation
+			`Out of nowhere, the mysterious whispers of the Rulei begin to seep into your mind once more. They murmur slowly at first, but grow faster and louder from the direcetion of the ancient vessel, eventually coalescing into one spoken phrase.`
+			`	"Beware.`
+			`	"Umbral's Reach."`
+			`	You feel tremendous weight behind the expression, as if it shook your mind itself. It's as though every emotion has flooded into your head at once- sadness, anger, joy, disgust, even some you can't seem to understand. Before you can respond to them, the voices quickly dissipate, and you're left alone in an uncomfortable silence. You aren't sure what "Umbral's Reach" is, but it seems the Rulei don't want you to be involved with it.`
```
Comment (Anarchist2) [inline data/rulei/rulei.txt:107]:
```suggestion
			`	You feel tremendous weight behind the expression, as if it shook your mind itself. It's as though every emotion has flooded into your head at once -- sadness, anger, joy, disgust, even some you can't seem to understand. Before you can respond to them, the voices quickly dissipate, and you're left alone in an uncomfortable silence. You aren't sure what "Umbral's Reach" is, but it seems the Rulei don't want you to be involved with it.`
```

File: data/rulei/rulei.txt (hunk 1 above)
Comment (Saugia) [reply data/rulei/rulei.txt:37]: Good question.

File: data/rulei/rulei.txt (hunk 3 above)
Comment (Saugia) [reply data/rulei/rulei.txt:102]: Good point.

File: data/rulei/rulei.txt [hunk 5]
```diff
@@ -90,6 +90,23 @@ mission "Rulei Planet"
...
+		never
+	to offer
+		has "First Contact: Rulei: offered"
+		has "visited system: L-118"
+	on enter "Kanguwa"
+		conversation
+			`Out of nowhere, the mysterious whispers of the Rulei seep into your mind once more. They murmur slowly at first, but grow faster and louder from the direction of the ancient vessel, eventually coalescing into one spoken phrase.`
+			`	"Beware..."`
+			`	"...Umbral's Reach."`
+			`	You feel tremendous weight behind the expression, as if it shook your mind itself. It's as though every emotion has flooded into your head at once -- sadness, anger, joy, disgust, even some you can't seem to understand. Before you can respond to them, the voices quickly dissipate, and you're left alone in an uncomfortable silence. You aren't sure what "Umbral's Reach" is, but it seems the Rulei don't want you to be involved with it.`
```
Comment (Anarchist2) [inline data/rulei/rulei.txt:107]:
```suggestion
			`	You feel tremendous weight behind the expression, as if it shook your mind itself. It's as though every emotion has flooded into your head at once - sadness, anger, joy, disgust, even some you can't seem to understand. Before you can respond to them, the voices quickly dissipate, and you're left alone in an uncomfortable silence. You aren't sure what "Umbral's Reach" is, but it seems the Rulei don't want you to be involved with it.`
```

File: data/rulei/rulei.txt [hunk 6]
```diff
@@ -90,6 +90,23 @@ mission "Rulei Planet"
...
 
+
+mission "Umbral's Reach"
+	invisible
+	landing
+	to complete
+		never
+	to offer
+		has "First Contact: Rulei: offered"
+		has "visited system: L-118"
```
Comment (Anarchist2) [inline data/rulei/rulei.txt:101]: Why does this system need to be visited?

File: data/rulei/rulei.txt [hunk 7]
```diff
@@ -90,6 +90,23 @@ mission "Rulei Planet"
...
+	on enter "Kanguwa"
+		conversation
+			`Out of nowhere, the mysterious whispers of the Rulei seep into your mind once more. They murmur slowly at first, but grow faster and louder from the direction of the ancient vessel, eventually coalescing into one spoken phrase.`
+			`	"Beware..."`
+			`	"...Umbral's Reach."`
```
Comment (Anarchist2) [inline data/rulei/rulei.txt:106]: Does this have something to do with the system Umbral (connected to Tarazed), or are they unrelated?

File: data/rulei/rulei.txt (hunk 6 above)
Comment (Saugia) [reply data/rulei/rulei.txt:101]: This system is the first system you pass through in the dark region area (and is the only way you can get through to it). The point of this mission is the vyu-Ir is able to tell you visited the area, and has a comment on it. Realistically, the player would have gone beyond L-118, but L-118 is still one of the Reach's systems.

File: data/rulei/rulei.txt (hunk 7 above)
Comment (Saugia) [reply data/rulei/rulei.txt:106]: No. "Umbral Reach" is the name for the dark region that was decided after a bit of discussion with others. Void, Abyss, etc. will be used in Mata content as far as I know, and anything else we came up didn't really fit it. Perhaps I reword this back to "Umbral Reach" so it doesn't possibly imply that "Umbral is reaching", as in the planet?

Comment (Anarchist2) [reply data/rulei/rulei.txt:106]: That sounds good.
```suggestion
			`	"...Umbral Reach."`
```

File: data/rulei/rulei.txt [hunk 8]
```diff
@@ -90,6 +90,23 @@ mission "Rulei Planet"
...
+		never
+	to offer
+		has "First Contact: Rulei: offered"
+		has "visited system: L-118"
+	on enter "Kanguwa"
+		conversation
+			`Out of nowhere, the mysterious whispers of the Rulei seep into your mind once more. They murmur slowly at first, but grow faster and louder from the direction of the ancient vessel, eventually coalescing into one spoken phrase.`
+			`	"Beware..."`
+			`	"...Umbral Reach."`
+			`	You feel tremendous weight behind the expression, as if it shook your mind itself. It's as though every emotion has flooded into your head at once - sadness, anger, joy, disgust, even some you can't seem to understand. Before you can respond to them, the voices quickly dissipate, and you're left alone in an uncomfortable silence. You aren't sure what "Umbral's Reach" is, but it seems the Rulei don't want you to be involved with it.`
```
Comment (Anarchist2) [inline data/rulei/rulei.txt:107]: The last sentence doesn't add anything.
```suggestion
			`	You feel tremendous weight behind the expression, as if it shook your mind itself. It's as though every emotion has flooded into your head at once - sadness, anger, joy, disgust, even some you can't seem to understand. Before you can respond to them, the voices quickly dissipate, and you're left alone in an uncomfortable silence.`
```
